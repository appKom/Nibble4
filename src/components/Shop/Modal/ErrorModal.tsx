import React, { FC } from "react";
import styled from "styled-components";

const CompleteModal: FC = () => {
  return (
    <Container>
      <h3> En feil oppsto! </h3>
      <p> Vennligst logg ut, og pr√∏v igjen</p>
    </Container>
  );
};

export default CompleteModal;

const Container = styled.div`
  text-align: center;
`;
