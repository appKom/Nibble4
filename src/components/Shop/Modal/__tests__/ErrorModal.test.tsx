import React from "react";
import { render, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import ErrorModal from "../ErrorModal";

afterEach(cleanup);

it("Should match snapshot", () => {
  const { asFragment } = render(<ErrorModal />);
  expect(asFragment()).toMatchSnapshot();
});
