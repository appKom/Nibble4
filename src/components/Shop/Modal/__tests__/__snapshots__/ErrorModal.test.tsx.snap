// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-bdfBwQ hMdVtH"
  >
    <h3>
       En feil oppsto! 
    </h3>
    <p>
       Vennligst logg ut, og pr√∏v igjen
    </p>
  </div>
</DocumentFragment>
`;
